14 create documentation files (#16)
AutoResearch/autora-template-cookiecutter: README.md

> # AutoRa Cookiecutter Template

Reviewer: ```suggestion
# AutoRA Cookiecutter Template
```

> This is a command line tool to create a template for a AutoRa package. 

Reviewer: ```suggestion
This is a command line tool for creating the structure of a new AutoRA subpackage. 
```

> [Here](https://autoresearch.github.io/autora/), you can find the documentation for AutoRa.

Reviewer: ```suggestion
[Here](https://autoresearch.github.io/autora/) you can find the documentation for AutoRA.
```

## Usage

First install the cookiecutter package:
```shell
pip install cookiecutter
```

to create the project, run 
```shell
cookiecutter https://github.com/AutoResearch/autora-template-cookiecutter
```

> You will be prompted with a series of questions to customize your project. If the prompts to these questions is `N/A - Press Enter to Skip`, you can skip the question.

Reviewer: ```suggestion
You will be prompted with a series of questions to customize your project. If a prompt displays, `N/A - Press Enter to Skip`, you can skip the question.
```
